addons/mail/security/ir.model.access.csv

```diff
@@ -71,3 +71,5 @@ access_discuss_gif_favorite,discuss.gif.favorite,model_discuss_gif_favorite,base
 access_discuss_voice_metadata_user,discuss.voice.metadata.user,model_discuss_voice_metadata,,0,0,0,0
 access_mail_partner_device,access_mail_partner_device,mail.model_mail_partner_device,base.group_user,0,0,0,0
 access_mail_notification_web_push,access_mail_notification_web_push,mail.model_mail_notification_web_push,base.group_user,0,0,0,0
+access_mail_message_translation_system,mail.message.translation,model_mail_message_translation,base.group_system,1,1,1,1
+access_mail_message_translation_user,mail.message.translation,model_mail_message_translation,base.group_user,0,0,0,0

```

Comment:
I believe the 0 rules lines are not necessary, but if they are, there should be one for no group (public, etc) and not just one for `base.group_user`.